### PR TITLE
Removed unused security rules fields.

### PR DESCRIPTION
**Pull request by seriva**
Cleanup for the documentation task #1322 regarding changing/adding security rules:

- Removes unused fields from the defaults for both Azure and AWS
- Fixes issue with duplicate security rules in AWS.

AWS security rule definition uses the fieldnames as for Azure. Ideally in the future we should match them to the origninal Terraform, templates. For now I left it as is to make it backwards compatible with older version of Epiphany.

### CODE REVIEW

**Review by pyrkamarcin — DISMISSED** [review]

**Review by pyrkamarcin — DISMISSED** [review]

File: core/src/epicli/cli/engine/providers/aws/InfrastructureBuilder.py [hunk 1]
```diff
@@ -84,7 +84,9 @@ def run(self):
             autoscaling_group = self.get_autoscaling_group(component_key, component_value, subnets_to_create, asg_index)
 
             for security_group in security_groups_to_create:
-                security_group.specification.rules += autoscaling_group.specification.security.rules
+                for rule in autoscaling_group.specification.security.rules:
+                    if self.rule_exists_in_list(security_group.specification.rules, rule) == False:
```
Comment (atsikham) [inline core/src/epicli/cli/engine/providers/aws/InfrastructureBuilder.py:88]: May be `if not`?

File: core/src/epicli/data/aws/terraform/infrastructure/security-group.j2 [hunk 2]
```diff
@@ -15,33 +15,30 @@ resource "aws_security_group" "{{ specification.name }}" {
...
+    cidr_blocks = ["{{ rule.source_address_prefix }}"]
+  {%- else %}
+    cidr_blocks = ["{{ rule.destination_address_prefix }}"]
+  {%- endif %}
+  }
 
-}
 {%- endfor %}
-
 }
```
Comment (atsikham) [inline core/src/epicli/data/aws/terraform/infrastructure/security-group.j2:44]: Newline

**Review by atsikham — CHANGES_REQUESTED** [review]
In general some day I would like to see code formatted according to PEP8. Not sure if vscode can do this, but it's for sure possible in pycharm.

LGTM, but a few changes requested.

File: core/src/epicli/cli/engine/providers/aws/InfrastructureBuilder.py (hunk 1 above)
Comment (seriva) [reply core/src/epicli/cli/engine/providers/aws/InfrastructureBuilder.py]: Fixed

File: core/src/epicli/data/aws/terraform/infrastructure/security-group.j2 (hunk 2 above)
Comment (seriva) [reply core/src/epicli/data/aws/terraform/infrastructure/security-group.j2:43]: fixed

**Comment by seriva** [comment]
Yes we can set up PEP8 in VSCode easily, I am pretty sure its even in the backlog somewhere.

**Review by atsikham — APPROVED** [review]

**Review by erzetpe — APPROVED** [review]

**Pull request merged**